Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -35,24 +35,10 @@ ARedemptionPlayer::ARedemptionPlayer()
 	StaminaRegenerationRate = 15.0f; // per second while not sprinting
 	bCanSprint = true;
 
-
-	if (UWidgetComponent* WidgetComponent = HealthComp->GetWidgetComp())
+	static ConstructorHelpers::FObjectFinder<UInputAction> SprintActionFinder(TEXT("/Game/Player/Input/Actions/IA_Sprint"));
```

Comment:
Why is this static?

Reply:
it will be created only once per execution of the game.